modules/logrotate/templates/logrotate.conf.erb

```diff
@@ -43,6 +43,16 @@
     # Remove empty .1 log files. There seems to be an issue where logrotate
     # will refuse to rotate the logs if an empty .1 log file exists.
     find <%= @matches %>* -wholename '/var/log/*' -name '*.1' -size 0 -delete
+    # Remove files bigger than $SIZE in chronological order while use of  
+    # disk space on the logging partition is greater than $MAX_USE
+    SIZE='300M'
+    MAX_USE=85
```

Comment:
Nice one putting this together! 👏 Can you add a bit more detail to the commit message? I think it's missing:

1) why we're making this change (ci_master machines running out of space)
2) where this gets applied (all machines? Only certain classes?)
3) why we chose a 300M threshold (e.g. "only n% of the logs are above 300M, but account for y% of the disk space")

Reply:
Commit message amended.